### ISSUE TITLE
Improve Markdown Rendering

### ISSUE THREAD

**Issue by Rud156** [git-touch/git-touch#132]
Opening Repos with a lot of Markdown content in its Readme.md file crashes the app.

Examples:
- https://github.com/transloadit/uppy
- https://github.com/mdn/content
- https://github.com/openpgpjs/openpgpjs

**Comment by pd4d10** [comment]
Thanks for the feedback.

Could you please provide more device info, for example, Android or iOS, System version?

**Comment by dhammel** [comment]
@pd4d10 I am having the same issue, on latest fdroid build 1.11.1.

Device: Google Pixel 4 non-xl, Android 11 Janurary 2021 security update. 

The other application, OctoDroid which I was looking to switch from as this integrates other services like Gitlab, also had similar issue was was addressed in an earlier update.

**Comment by linsui** [comment]
I have the same issue. How about setting the flutter render engine as default before this issue is properly solved?

**Comment by pd4d10** [comment]
Yep, it seems that the Android webview widget may have some problems, for example #123 and #194

> setting the flutter render engine as default

Makes sense. We could make Flutter render engine as default on Android.

**Comment by pd4d10** [comment]
The reason we prefer Webview is that inline HTML is not supported in the Flutter markdown widget.

BTW, this issue could be submitted to the official repo: https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3A%22p%3A+webview%22
